client/src/modules/dashboard/Chapters/components/ChapterForm.tsx

```diff
@@ -16,20 +17,65 @@ interface ChapterFormProps {
   submitText: string;
 }
 
-type Fields =
-  | [keyof ChapterFormData, boolean, boolean]
-  | [keyof ChapterFormData, boolean]
-  | [keyof ChapterFormData];
+type Fields = {
+  key: keyof ChapterFormData;
+  placeholder: string;
+  label: string;
+  required: boolean;
+  type: string;
+};
 
 const fields: Fields[] = [
-  ['name', true],
-  ['description', true],
-  ['details', true],
-  ['city', true],
-  ['region', true],
-  ['country', true],
-  ['category', true],
-  ['imageUrl', true],
+  {
+    key: 'name',
+    label: 'Chapter name',
+    placeholder: 'freeCodeCamp',
+    required: true,
+    type: 'text',
+  },
+  {
+    key: 'description',
+    label: 'Description',
+    placeholder:
+      'freeCodeCamp is a non-profit organization that helps people learn to code for free',
+    required: true,
+    type: 'textarea',
+  },
+  {
+    key: 'city',
+    label: 'City',
+    placeholder: 'San Fransisco',
+    required: true,
+    type: 'text',
+  },
+  {
+    key: 'region',
+    label: 'Region',
+    placeholder: 'California',
+    required: true,
+    type: 'text',
+  },
+  {
+    key: 'country',
+    label: 'Country',
+    placeholder: 'United States of America',
+    required: true,
+    type: 'text',
+  },
+  {
+    key: 'category',
+    label: 'Category',
+    placeholder: 'Education and nonprofit work',
+    required: true,
+    type: 'text',
+  },
+  {
+    key: 'imageUrl',
+    label: 'Image Url',
+    placeholder: 'www.freecodecamp.com',
```

Comment:
```suggestion
    placeholder: 'www.freecodecamp.org',
```
Nitpick, .com just redirects to .org